### PR TITLE
Add restaurant, global promo management

### PR DESCRIPTION
**Pull request by AdityaA1998**

### CODE REVIEW

File: src/components/Manager/ManageRestaurant.vue [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+                        );
+                    }
+                ]
+            }
+        }),
+        methods: {
+            async getRestaurants() {
+                const res = await axios.get(`/manager/get-restaurants/`);
+                if (res.status == 200 || res.status == 304) {
+                    console.log(res.data)
```
Comment (jelneo) [inline src/components/Manager/ManageRestaurant.vue:145]: remove

File: src/components/Manager/ManageRestaurant.vue [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+                    this.restList = res.data;
```
Comment (jelneo) [inline src/components/Manager/ManageRestaurant.vue:146]: need to convert the min spend from cents to dollars

File: src/components/Manager/ManageRestaurant.vue [hunk 3]
```diff
@@ -0,0 +1,240 @@
+<template>
+    <v-container>
+        <h2 style="color: orange; padding: 50px 50px"> Manager Restaurants</h2>
```
Comment (jelneo) [inline src/components/Manager/ManageRestaurant.vue:3]: Manage*

File: src/components/Manager/CustomerOrderSummary.vue [hunk 4]
```diff
@@ -65,6 +65,7 @@
                             let res = this.orderInfo.filter(i => i.year == this.selectedYear);
                             res.forEach(element => {
                                 element.month = this.months[element.month - 1];
+                                element.totalprice = parseInt(element.totalprice).toLocaleString("en-SG");
```
Comment (whenzei) [inline src/components/Manager/CustomerOrderSummary.vue:68]: total price is in cents, please change it to dollars

**Comment by whenzei** [comment]
The html element <up></up> in ManagerLanding is causing error in the console.

**Pull request merged**